sklearnex/tests/test_run_to_run_stability.py

```diff
@@ -148,8 +151,13 @@ def _run_test(estimator, method, datasets):
 def test_standard_estimator_stability(estimator, method, dataframe, queue):
     if estimator in ["LogisticRegression", "TSNE"]:
         pytest.skip(f"stability not guaranteed for {estimator}")
-    if estimator in ["KMeans", "PCA"] and method == "score" and queue == None:
+    if estimator in ["KMeans", "PCA"] and "score" in method and queue == None:
         pytest.skip(f"variation observed in {estimator}.score")
+    if estimator in ["IncrementalEmpiricalCovariance"] and method == "mahalanobis":

```

Comment:
```suggestion
    if estimator is "IncrementalEmpiricalCovariance" and method == "mahalanobis":
```
Unless its not functionally equivalent - is regular EmpiricalCovariance not part of this?

Reply:
good catch. EmpiricalCovariance will have the same issue when it comes out of preview

Reply:
Sounds good, that makes sense. Or could be super sneaky and use `if estimator in "IncrementalEmpiricalCovariance"` to cover both 😏 (don't actually do this)